src/preact/utils.js

```diff
@@ -16,15 +16,31 @@ export function useResourcesNotify() {
 }
 
 /**
- * Combines multiple refs to pass into `ref` prop.
- * @param {...any} refs
- * @return {function(!Element)}
+ * @param {any} ref
+ * @param {!Element} value
  */
-export function refs(...refs) {
-  return (element) => {
-    for (let i = 0; i < refs.length; i++) {
-      const ref = refs[i];
-      typeof ref == 'function' ? ref(element) : (ref.current = element);
+function setRef(ref, value) {
+  if (typeof ref === 'function') {
+    ref(value);
+  } else if (ref) {
+    ref.current = value;
+  }
+}
+
+/**
+ * Combines refs to pass into `ref` prop.
+ * @param {any} refOne
+ * @param {any} refTwo
+ * @return {function()}
+ */
+export function useMergeRefs(refOne, refTwo) {
+  return useMemo(() => {
+    if (refOne === null && refTwo === null) {
+      return null;
     }
-  };
+    return (element) => {
```

Comment:
nit: Seems even easier to take an array

```js
export function useMergeRefs(refs) {
  return useCallback(
    (element) => refs.forEach((ref) => setRef(ref, element)),
    refs
  );
}
```

Reply:
Updated, except used a native for loop since @jridgewell is not a fan of `forEach` 🙂

Reply:
you learn fast 😆